app/javascript/tests/patient/Patient.test.js

```diff
@@ -126,9 +126,11 @@ describe('Patient', () => {
   it('Properly renders contact information section when patient is a minor', () => {
     const wrapper = shallow(<Patient details={mockPatient5} hoh={mockPatient1} collapse={true} edit_mode={false} jurisdiction_paths={mockJurisdictionPaths} />);
     const section = wrapper.find('#contact-information');
+    const HOHLinkText = 'View contact info for Head of Household:' + mockPatient1.first_name + ' ' + mockPatient1.middle_name + ' ' + mockPatient1.last_name;
```

Comment:
nitpick but can we call this `hohLinkText`

Reply:
or honestly just do it inline, we have a lot of tests with long text checks

Reply:
Can you do template strings here?
```
`View contact info for Head of Household: ${mockPatient1.first_name} ${mockPatient1.middle_name} ${mockPatient1.last_name}\`;
```